examples/native/components/ControlExamples/Radio/Radio.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react-native';
+import { Radio } from './Radio';
+
+const radio_stations = ['104.8MHz', '909 kHz', '90FM'];
+
+const RadioMeta: ComponentMeta<typeof Radio> = {
+  title: 'Radio control',
+  component: Radio,
+  args: {
+    selection: radio_stations[0],
+  },
+  argTypes: {
+    selection: {
+      options: radio_stations,
+      control: { type: 'radios' },
```

Comment:
Wouldn't do anything in the scope of this PR, but longer-term we could consider aligning the control types with the control types of the Web storybook: https://storybook.js.org/docs/react/essentials/controls#annotation

They seem to have a bit different setup with the controls, especially with the `options` ones. Being uniform could help via e.g. allowing us to point to the same docs and making the library less confusing to use. This would be a longer-term V6 target though, not an Alpha 1 target.

![Screen Shot 2021-07-17 at 10 33 45](https://user-images.githubusercontent.com/6605505/126029879-277080c8-eddf-4a9b-80f4-6a2519b2e8d9.png)

Reply:
Yeah I totally agree with you on this, mostly at this stage I just wanted to release something that would replace knobs and get the fundamentals of controls in. 

I made an issue for all the remaining controls work and part of that is basically  getting it closer to the web implementation.